sycl/test/basic_tests/vec_convert_f_to_f.cpp

```diff
@@ -0,0 +1,62 @@
+// XFAIL: cuda
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=CPU %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
// RUN: %CPU_RUN_PLACEHOLDER %t.out
```
Duplicate run on CPU here.